src/test/java/org/jpeek/MainTest.java

```diff
@@ -57,18 +57,30 @@ public void createsXmlReports() throws IOException {
         ).affirm();
     }
 
-    @Test(expected = ParameterException.class)
+    @Test
     public void crashesIfInvalidInput() throws IOException {
-        Main.main("hello");
+        new Assertion<>(
+            "Must throw an exception if parameter is invalid",
+            () -> {
+                Main.main("hello");
+                return null;
+            }, new Throws<>(ParameterException.class)
+        ).affirm();
     }
 
-    @Test(expected = IllegalStateException.class)
+    @Test
```

Comment:
@HDouss You can remove the throws clause from the test now

Reply:
@paulodamaso it is not possible here, as `Files.createTempDirectory` throws `IOException`.